boards/shields/lcd_par_s035/Kconfig.shield

```diff
@@ -0,0 +1,5 @@
+# Copyright 2024 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+config SHIELD_LCD_PAR_S035
```

Comment:
It does not seem to me that this Kconfig would be set when building for the mcx board with this shield, IE with:
`west build -p always -b frdm_mcxn947//cpu0 samples/drivers/display -DSHIELD='lcd_par_s035_8080'`.

Maybe we should add another Kconfig to check for the `lcd_par_s035_8080` shield name?

Reply:
Thanks, I have update the `Kconfig`